README.md

```diff
@@ -62,6 +62,8 @@ composer require --dev aeliot/php-cs-fixer-baseline
 | b          | baseline  | Name of baseline file                  | .php-cs-fixer-baseline.json |
 | c          | config    | Name of config file                    | .php-cs-fixer.dist.php      |
 | f          | finder    | Name of file with definition of Finder | .php-cs-fixer-finder.php    |
+| d          | dir       | Config files path                      | ``''``                      |
```

Comment:
Add option in alphabetical order. It makes documentation more readable and helps to avoid merge conflicts